docs/source/scancode-toolkit/cli-reference/core-options.rst

```diff
@@ -0,0 +1,282 @@
+`Core` Options
+==============
+
+.. include::  /scancode-toolkit/rst_snippets/basic_options.rst
```

Comment:
Is it normal to have absolute slash-prefixed paths there?

Reply:
It is not. Even `doc8` gives an error when checking for doc standards here. I thought it'd be better if all the snippets are in one location. If you prefer otherwise I'll change that.
[Note]
Some snippets are used cross-section but a majority of them are not. For example,
`rst_snippets/custom_output_format.rst` is included both in `cli-referance/output-format.rst` and `tutorials/how_to_format_scan_output.rst`. What should I do in these cases?

Reply:
I don't think there is a problem with the location of the snippets, per-say. I think @pombredanne just means we should use relative paths if possible? Not sure how sphinx is configured in this case, but changing to `./scancode-toolkit/rst_snippets/basic_options.rst` might be enough here?

Reply:
That'd give an error `InputError: [Errno 2] No such file or directory: 'source/scancode-toolkit/cli-reference/scancode-toolkit/rst_snippets/post_scan_options.rst'.` as in sphinx using `./` relates to the rst_snippets being located in the same directory as the file it's being included in. Right?